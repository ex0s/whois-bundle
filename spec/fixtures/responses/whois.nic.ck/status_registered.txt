
domain:      google.co.ck
descr:       Google Inc
descr:       1600 Amphitheatre Parkway,
descr:       Mountain View, CA 94043,
descr:       United States
admin-c:     MM15-CKNIC
tech-c:      DA7-CKNIC
zone-c:      ODA1-CKNIC
billing-c:   MM15-CKNIC
nserver:     ns1.google.com
nserver:     ns2.google.com
remarks:     expires 20131101
mnt-by:      CK-NIC-MNT
changed:     marion@telecom.co.ck 20111004
source:      CKNIC

role:        Oyster DNS Admin
address:     Telecom Cook Islands Ltd.
address:     P.O. Box 106
address:     Rarotonga
address:     CK
e-mail:      soa@oyster.net.ck
admin-c:     RM1-CKNIC
tech-c:      RM1-CKNIC
nic-hdl:     ODA1-CKNIC
mnt-by:      CK-NIC-MNT
changed:     robert@telecom.co.ck 19990420
source:      CKNIC

person:      MarkMonitor Inc
address:     PMB 155, 10400 Overland Road,
address:     Boise, ID,
address:     US 83709
phone:       +1 208 389 5740
fax-no:      +1 208 389 5799
e-mail:      ccops@MarkMonitor.com
nic-hdl:     MM15-CKNIC
mnt-by:      CK-NIC-MNT
changed:     joe@telecom.co.ck 20060116
source:      CKNIC

person:      Domain Administrator
address:     1600 Amphitheatre Parkway,
address:     Mountain View, CA 94043,
address:     United States
phone:       +1 650 253 0000
fax-no:      +1 650 618 8571
e-mail:      dns-admin@google.com
nic-hdl:     DA7-CKNIC
mnt-by:      CK-NIC-MNT
changed:     joe@telecom.co.ck 20060719
source:      CKNIC


