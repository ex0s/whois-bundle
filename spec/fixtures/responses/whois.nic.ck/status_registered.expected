#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %ERROR{AttributeNotSupported}

#updated_on
  %s %CLASS{time}
  %s %TIME{2011-10-04}

#expires_on
  %s %CLASS{time}
  %s %TIME{2013-11-01}

#nameservers
  %s %CLASS{array}
  %s %SIZE{2}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns1.google.com"
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns2.google.com"
